steps/cadence-innovus-flowsetup/setup.tcl

```diff
@@ -110,9 +115,14 @@ if {[file exists $vars(adk_dir)/stdcells-bc.lib]} {
 # - Voltage: lowest
 # - Temperature: lowest (temperature inversion at 28nm and below)
 
+# FIXME/TODO all the globs (above and below) should be lsort'ed for better
+# determinacy; but I will save that for a separate branch fix on some later day...
```

Comment:
Can we not have this todo in the comments? Maybe turn it into an issue?

Reply:
done (i think it's already an issue btw)